modules/monitoring-applying-custom-alertmanager-configuration.adoc

```diff
@@ -55,6 +55,15 @@ receivers:
 <2> `<your_matching_rules>` specifies the target alerts.
 <3> `receiver` specifies the receiver to use for the alert.
 +
+[NOTE]
+====
+Use `matchers` as the key name to indicate the matchers that an alert has to fulfill to match the node.
+Do not use `match` or `match_re` as key names because both of these names are deprecated and will be removed in a future release.
+
+In addition, if you define inhibition rules, use `target_matchers` as the key name to indicate the target matchers and `source_matchers` to indicate the source matchers.
+Do not use `target_match`, `target_match_re`, `source_match`, or `source_match_re` as key names because these names are deprecated and will be removed in a future release.
```

Comment:
If you update lines 60-61, make parallel updates to lines 63-64.

Reply:
Done.